sched/test_ratio_static.py

```diff
@@ -0,0 +1,231 @@
+"""
+Ratio scheduler is fast and fair scheduler based on weighted round-robin
+principle. Functional test for Ratio scheduler requires intensive loads to
+evaluate how fair the load distribution is.
+"""
```

Comment:
One more copy & paste. The header is confusing and must describe the test, not the scheduler itself. I'm selecting check boxes in https://github.com/tempesta-tech/tempesta/issues/811 and it's hard to do because I have to identify what's different in the code from 3 other tests.